README.md

```diff
@@ -1,26 +1,30 @@
 # go-libp2p-loggables
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/project-libp2p-blue.svg?style=flat-square)](http://github.com/libp2p/libp2p)
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
 > logging helpers for go-libp2p
 
-## Documenation
+## Install
 
-See https://godoc.org/github.com/ipfs/go-libp2p-loggables.
+```
+go get github.com/libp2p/go-libp2p-loggables
+```
```

Comment:
It is library, you can't really install it but if we do this in other lib only repos too, then let's keep it.